### PR TITLE
add `telega-chat-skip-delete-confirmation-for` custom option

### PR DESCRIPTION
**Pull request by macovsky**
specifies Chat Temex for chats which can be deleted without typing confirmation

### CODE REVIEW

File: telega-customize.el [hunk 1]
```diff
@@ -215,6 +215,13 @@ Tracking notifications for telega buffers will use the
              (or unmuted mention))
   :group 'telega)
 
+(defcustom telega-skip-delete-confirmation-for nil
+  "*Specifies Chat Temex for chats which can be deleted without
```
Comment (macovsky) [inline telega-customize.el:219]: btw what this `*` is about? I just copied it from `telega-use-tracking-for`

File: docs/telega-ellit.org [hunk 2]
```diff
@@ -573,6 +573,7 @@ Use ~telega-story-match-p~ to match a story.
 - {{{user-option(telega-chat-group-messages-for, 2)}}}
 - {{{user-option(telega-chat-show-deleted-messages-for, 2)}}}
 - {{{user-option(telega-root-view-grouping-alist, 2)}}}
+- {{{user-option(telega-skip-delete-confirmation-for, 2)}}}
```
Comment (macovsky) [inline docs/telega-ellit.org:576]: do I have to generate the manual?

File: telega-customize.el (hunk 1 above)
Comment (zevlg) [reply telega-customize.el:219]: "*" Stands for "user might want to change default value".

Name it like `telega-chat-delete-skip-confirmation-for` (i.e. "telega-chat-" prefix matters) and put this custom option to `telega-chat` customization group.

**Review by zevlg — CHANGES_REQUESTED** [review]

File: docs/telega-ellit.org (hunk 2 above)
Comment (zevlg) [reply docs/telega-ellit.org:576]: > do I have to generate the manual?

No need, I'll rebuild docs

File: telega-customize.el (hunk 1 above)
Comment (macovsky) [reply telega-customize.el:219]: awesome, fixed!

**Comment by zevlg** [comment]
Merged in, thank you for contribution!

**Comment by macovsky** [comment]
> Merged in, thank you for contribution!

excellence! thanks for your thoughtful consideration as always—this new custom option is much better!

**Pull request merged**